pkg/apis/core/v1beta1/types_shoot.go

```diff
@@ -700,6 +700,20 @@ type KubeAPIServerConfig struct {
 	// Defaults to 1h.
 	// +optional
 	EventTTL *metav1.Duration `json:"eventTTL,omitempty" protobuf:"bytes,12,opt,name=eventTTL"`
+	// Logging contains configuration for the logs level and http access logs
+	// +optional
+	Logging *Logging `json:"logging,omitempty" protobuf:"bytes,13,opt,name=logging"`
+}
+
+// Logging contains configuration for the logs level and http access logs
+type Logging struct {
```

Comment:
Should we rename this (everywhere)?

```suggestion
type KubeAPIServerLogging struct {
```

Reply:
Are you sure that under `type KubeAPIServerConfig struct` the Logging part should be prefixed with `KubeAPIServer...`?
If you say yes, I will rename it.

Reply:
The struct is in `v1beta1` package, so its name should be more specific if possible. The field in the `KubeAPIServerConfig` struct using it can still be called `Logging`.